apps/frontend/src/app/backups-overview/component/information-panel/information-panel.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="card info">
+  <div class="card-block">
+    <div class="card-text">
+      <p>On this page, you can:</p>
+      <ul>
+        <li>
+          <strong>See Alerts</strong>: View the current status and details of
+          any alerts that have been triggered. Alerts are categorized by their
+          severity.
+        </li>
+        <li>
+          <strong>See all backups with basic data</strong>: Browse through a
+          list of all backups, including essential information such as Backup
+          ID, size and creation date.
```

Comment:
Consider changing "Backup ID" to "Saveset", because we don't show the "Backup ID" anymore

Reply:
The next PR which I will open as soon as this one is merged contains a completely new text for the information panel, which our POs wrote, so this will be changed